webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala

```diff
@@ -323,15 +323,18 @@ final class AsyncDeploymentActor(
 
         case Full(nodeIds) =>
           logger.info(s"Successful policy update '${id}' [started ${startTime.toString(timeFormat)} - ended ${endTime.toString(timeFormat)}]")
+          println("prout")
```

Comment:
:thinking:

Reply:
:boom:

Reply:
I think you did not want to commit that file and was automatically commited by rudder-dev